### DISCUSSION TITLE
Revert to V2

### DISCUSSION THREAD

**Discussion by wakmail** [noah-nuebling/mac-mouse-fix#797] [Questions]
How do you go back to Version 2? I accidentally updated the app to v3 and can't use the app anymore. Also, how do you save preferences when going from v3 to v2?

**Comment by noah-nuebling** [comment]
Hey there @wakmail thanks for reaching out.

If you want to go back, you can just delete MMF 3 and reinstall the latest MMF 2 version (Currently [2.2.3](https://github.com/noah-nuebling/mac-mouse-fix/releases/tag/2.2.3)). 

If you have trouble enabling the app after downgrading, you might want to delete MMF 3 and empty the trash and then restart your computer. 

To save your MMF 3 settings, open the About tab and click the MMF Logo while holding Command and Shift on your keyboard.
<img width="556" alt="Screenshot 2024-01-17 at 02 34 47" src="https://github.com/noah-nuebling/mac-mouse-fix/assets/40808343/5e00ee67-581f-4d2a-9657-4f00aed1e0dc">

This will reveal a file called **config.plist**, which contains all your settings. You can save it somewhere to and then copy it back later to restore your settings.

If you tell me more details about why Mac Mouse Fix 3 isn't working on your end, I will try to find a fix for that.

I hope that helps! Let me know if you have any questions.

**Comment by wakmail** [comment]
MM3 works great! I just don't have a credit or debt card. Right now I use MM2 just for the middle click gestures and MOS for the scrolling. One suggestion for MM3 is more options for Click-Dragging the middle button since you can do a Click-Drag, Double Click-Drag, and Triple Click-Drag, but there are only two options currently. Also, you should implement the ability to do a keyboard shortcut for all customizable buttons.
